pipelines/internal/commands/run/run.go

```diff
@@ -335,13 +335,11 @@ func buildRequest(filename, project string) (*genomics.RunPipelineRequest, error
 	}
 
 	if filename != "" {
-		if err := parseJSON(filename, &actions); err != nil {
```

Comment:
I am just trying to understand the exact culprit, it seems here instead of appending to actions we overwrite it. Right?

Reply:
Correct